docs/README.md

```diff
@@ -62,6 +62,23 @@ externally hosted documentation. The reference will look like `integrate_connect
 
 Replace <connector_name> with the name of your connector, e.g., `elasticsearch` for `flink-connector-elasticsearch`.
 
+### Include external documentation hosted on forks
+
+The configuration above works for connectors already merged in `apache/flinl-connector-<connector-name>` repos.
+When contributing to a Flink's connector, the documentation is hosted in a fork of one of these repos.
+
+To preview any connector docs changes not yet merged follow these steps:
+
+1. (If necessary) Push the latest changes to the connector docs to the forked repo, in a branch
```

Comment:
> docs to the forked repo

It is not clear which fork this is, but now I guess you are talking about the contributors personal fork? Can you make this clearer

Reply:
That is exactly the point. The goal is enabling contributors to preview their changes while they are still in their personal fork, maybe even before raising a PR